core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -522,19 +522,21 @@ class GroupCoordinatorTest extends JUnitSuite {
 
   @Test
   def staticMemberRejoinWithFollowerIdAndChangeOfProtocol() {
-    val rebalanceResult = staticMembersJoinAndRebalance(leaderInstanceId, followerInstanceId)
+    val rebalanceResult = staticMembersJoinAndRebalance(leaderInstanceId, followerInstanceId, sessionTimeout = DefaultSessionTimeout * 2)
```

Comment:
Ditto, why double the value here?

Reply:
Doubling is aiming to reproduce the scenario where `rebalanceTImeout` < `sessionTimeout`. I could write comments for both scenarios.